### PR TITLE
[one-cmds] Improve help message of one-quantize granularity

### PR DESCRIPTION
**Pull request by zetwhite**
This PR improve one-quantize granularity options, by adding a word 'weight'.

ONE-DCO-1.0-Signed-off-by: SeungHui Youn <sseung.youn@samsung.com>

related issue : https://github.com/Samsung/ONE/issues/12081

### CODE REVIEW

**Comment by seanshpark** [comment]
FYI, need to patch `compiler/one-cmds/one-quantize` too. (in another PR)
and may be `compiler/luci/pass/src/helpers/Strings.cpp` too ?

File: compiler/one-cmds/one-quantize [hunk 1]
```diff
@@ -94,7 +94,7 @@ def _get_parser():
     quantization_group.add_argument(
         '--granularity',
         type=str,
-        help='quantization granularity (supported: layer, channel, default=layer)')
+        help='weight quantization granularity (supported: layer, channel, default=layer)')
```
Comment (jinevening) [inline compiler/one-cmds/one-quantize:97]:
```suggestion
        help='weight quantization granularity (supported: layer, channel, default=layer). activation is quantized per layer.')
```

Comment (zetwhite) [reply compiler/one-cmds/one-quantize]: updated!

**Comment by zetwhite** [comment]
> FYI, need to patch `compiler/one-cmds/one-quantize` too. (in another PR)

I failed to catch this sentence. 
I thought this PR patch `one-cmds/one-quantize` :thinking:

**Comment by seanshpark** [comment]
> I thought this PR patch one-cmds/one-quantize

Yes, this patch if for that. But there are same messages in other places that need changes too.

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**